e2e/test/iothub/twin/TwinE2ETests.cs

```diff
@@ -581,4 +583,12 @@ await deviceClient
             Assert.IsFalse(serviceTwin.Properties.Reported.Contains(propName1));
         }
     }
+
+    internal class CustomTwinProperty
```

Comment:
I think you left out some parts of the change from this commit.

Reply:
umm, what do you mean?

Reply:
ignore me. I totally missed out on the static property you added that uses this class.